<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entry</class>
 <widget class="QWidget" name="Entry">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chkChecked"/>
     </item>
     <item>
      <widget class="QLabel" name="lblIcon">
       <property name="pixmap">
        <pixmap>../../../../../../usr/share/icons/oxygen/32x32/categories/applications-internet.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="vline">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTitle">
         <property name="text">
          <string>&lt;b&gt;Mon paquet&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblShortDesc">
         <property name="text">
          <string>Ceci est un joli paquet avec une belle ic√¥ne.</string>
         </property>
         <property name="indent">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblStar1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../../usr/share/icons/oxygen/22x22/actions/rating.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblStar2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../../usr/share/icons/oxygen/22x22/actions/rating.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblStar3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../../usr/share/icons/oxygen/22x22/actions/rating.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblStar4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../../usr/share/icons/oxygen/22x22/actions/rating.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblStar5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../../usr/share/icons/oxygen/22x22/actions/rating.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblDownload">
         <property name="text">
          <string>1,0 Mio / 1,34 Mio</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
